source/kraken/configuration.cpp

```diff
@@ -76,6 +76,7 @@ po::options_description get_options_description(const boost::optional<std::strin
         ("GENERAL.log_level", po::value<std::string>(), "log level of kraken")
         ("GENERAL.log_format", po::value<std::string>()->default_value("[%D{%y-%m-%d %H:%M:%S,%q}] [%p] [%x] - %m %b:%L  %n"), "log format")
 
+        ("GENERAL.enable_deadline", po::value<bool>()->default_value(true), "enforce deadline of request")
```

Comment:
enable_deadline is a little bit generic... enable_message_deadline? or enable_zeromq_deadline? enable_zeromq_ttl?

Reply:
what about `enable_request_deadline` ?